### PR TITLE
Use FreeResponse translations

### PR DESCRIPTION
**Pull request by Hamms**
Use translations of FreeResponse fields (added in https://github.com/code-dot-org/code-dot-org/pull/33970) in the view. Also remove the placeholder placeholder we added to hold us over until now.

## Links

- [jira](https://codedotorg.atlassian.net/browse/FND-984)

## Testing story

Added a test translation file to `dashboard/config/locales/free_responses.es-MX.yml`:

```yml
---
es-MX:
  data:
    placeholder:
      csp_socialBelonging_intervention_2018: "yay, a translation"
```

Then checked `http://localhost-studio.code.org:3000/s/csp1-2018/stage/1/puzzle/3/lang/es-mx`:

image

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: dashboard/app/views/levels/_free_response.html.haml [hunk 1]
```diff
@@ -48,8 +48,7 @@
       ReactDOM.render(React.createElement(dashboard.Attachments, {showUnderageWarning: !appOptions.is13Plus, readonly: #{!!@view_options.readonly_workspace}}), document.querySelector('#free-response-upload'));
 
   - height = level.height || '80'
-  -# FND-984 placeholder needs to be localized level property.
-  - placeholder = level.placeholder || I18n.t('free_response.placeholder')
```
Comment (daynew) [inline dashboard/app/views/levels/_free_response.html.haml:52]: I think we still want the `|| I18n.t('free_response.placeholder')` so the default text is still there. Most levels don't define the 'placeholder' property.

File: dashboard/config/locales/en.yml [hunk 2]
```diff
@@ -719,8 +719,6 @@ en:
     help_support: "Help and support"
     documentation: "Documentation"
     tutorials: "Tutorials"
-  free_response:
-    placeholder: "Enter your answer here"
```
Comment (daynew) [inline dashboard/config/locales/en.yml]: Related to comment above. I'm not sure we want to remove this.

**Review by daynew — APPROVED** [review]
Looks good except I could you double check my comment about removing the default placeholder text?

File: dashboard/app/views/levels/_free_response.html.haml (hunk 1 above)
Comment (Hamms) [reply dashboard/app/views/levels/_free_response.html.haml:52]: oh, good catch!

**Review by daynew — APPROVED** [review]

**Pull request merged**